src/util/bip143.rs

```diff
@@ -19,15 +19,16 @@
 //! signatures, which are placed in the scriptSig.
 //!
 
+use core::ops::{Deref, DerefMut};
+
+use crate::io;
+
```

Comment:
nit: We don't need this newline, right?

Reply:
I left `io` separate because its done like that elsewhere and so as to make the regrouping palatable to _everyone_ :) `rustfmt` will hopefully come in soon anyways, I just did a little tidy up so my OCD wasn't triggered in the later patches of the PR.